getting-started.ipynb

```diff
@@ -0,0 +1,591 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 5,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "Python 3 (ipykernel)",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.8.8"
+    },
+    "colab": {
+      "name": "getting-started.ipynb",
+      "provenance": []
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Getting Started"
+      ],
+      "metadata": {
+        "id": "HgljXEAJEcFq"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Welcome to **PyTorch-Ignite**’s quick start guide that covers the\n",
+        "essentials of getting a project up and running while walking through\n",
+        "basic concepts of Ignite. In just a few lines of code, you can get your\n",
+        "model trained and validated. The complete code can be found at the end\n",
+        "of this guide."
+      ],
+      "metadata": {
+        "id": "P9VjKOirEcFu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Prerequisites\n",
+        "\n",
+        "This tutorial assumes you are familiar with the:\n",
+        "\n",
+        "1.  Basics of Python and deep learning\n",
+        "2.  Structure of PyTorch code"
+      ],
+      "metadata": {
+        "id": "1QNvbg3SEcFw"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Installation\n",
+        "\n",
+        "1.  From `pip`\n",
+        "\n",
+        "``` shell\n",
+        "pip install pytorch-ignite\n",
+        "```\n",
+        "\n",
+        "1.  From `conda`\n",
+        "\n",
+        "``` shell\n",
+        "conda install ignite -c pytorch\n",
+        "```\n",
+        "\n",
+        "See [here](https://pytorch-ignite.netlify.app/docs/how-to-guides/installation) for other installation\n",
+        "options."
+      ],
+      "metadata": {
+        "id": "XTHzzYyoEcFy"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Code\n",
+        "\n",
+        "Import the following:"
+      ],
+      "metadata": {
+        "id": "DcnSr5sGEcFz"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "source": [
+        "import torch\n",
+        "from torch import nn\n",
+        "from torch.utils.data import DataLoader\n",
+        "from torchvision.datasets import MNIST\n",
+        "from torchvision.models import resnet18\n",
+        "from torchvision.transforms import Compose, Normalize, ToTensor\n",
+        "from tqdm import tqdm\n",
```

Comment:
Never used. 
However, the `ProgressBar` handler could be a nice feature to add, couldn't it ?

Reply:
Thanks for pointing that out. Added ProgressBar. However, I don't understand what you mean by 
> EDIT : Only 5 epochs, maybe let the code in a simple version should be more relevant.

Reply:
If you want to keep the code very simple, maybe ProgressBar is too much, especially regarding to the low number of epochs.

Reply:
Got it. We can introduce them later in tutorials as it isn't a major Ignite concept and I would like to keep this notebook as short as possible.